sstables/sstables.hh

```diff
@@ -283,11 +283,13 @@ public:
 
     // A reader which doesn't use the index at all. It reads everything from the
     // sstable and it doesn't support skipping.
+    using integrity_check = bool_class<class integrity_check_tag>;
```

Comment:
This should be put in the top level in `sstables.hh`, in the commit which introduces it, instead of moving it around.